Gemfile

```diff
@@ -0,0 +1,13 @@
+source "https://rubygems.org"
+
+group  :development, :test do
+	gem 'gauge-ruby', '~>0.2.0'
+	gem 'capypage', '~> 0.2.7'
+	gem 'test-unit', '~> 3.2.0'
+	gem 'selenium-webdriver', '~> 2.50.0'
+  gem 'process_builder', '~>1.1.0'
+  gem 'rest-client', '~>1.8.0'
+  gem 'os'
+  gem 'rake'
+
+end
```

Comment:
Please checkin Gemfile.lock, it contains specific versions of dependencies. No need to specify versions of gems here.

And indent this better :)